protobuf/flow/executiondata/executiondata.proto

```diff
@@ -230,4 +254,78 @@ message GetRegisterValuesResponse {
 
   // raw register values at the given height.
   repeated bytes values = 1;
-}
\ No newline at end of file
+}
+
+// The request for SubscribeAccountStatuses
+message SubscribeAccountStatusesRequest {
```

Comment:
Since we are introducing new API I suggest to enforce "having only `start_block_id` or `start_block_height`" in protobuf using [oneof](https://protobuf.dev/programming-guides/proto3/#oneof) keyword. This will make a cleaner API and simplify implementation.

Reply:
instead of using `oneof`, let's split the API into 3 methods like the polling endpoints. I made a similar request for the block streaming endpoints. e.g.
* `SubscribeAccountStatusesFromStartBlockID`
* `SubscribeAccountStatusesFromStartHeight`
* `SubscribeAccountStatusesFromLatestBlock`

This adds a bit of boiler plate, but simplifies the client and server side logic